File: java_tasks/begin5.js
let a = Number(prompt("Введите длину ребра куба"));

let V = Math.pow(a, 3);
let S = 6 * Math.pow(a, 2);

alert("Объём куба: " + V + "\n" +
    "Площадь поверхности куба: " + S);

File: java_tasks/begin5.java
package java_tasks;

import java.util.Scanner;

class JavaTasks_Begin5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Введите длину ребра куба: ");
        float a = scanner.nextFloat();

        float V = (float) Math.pow(a, 3);
        float S = 6 * (float) Math.pow(a, 2);

        System.out.println("Объём куба: " + V);
        System.out.println("Площадь поверхности куба: " + S);
        scanner.close();
    }
}
